### PR TITLE
Makes some properties read-only & remove Mu environment variables

### PR DESCRIPTION
**Pull request by cecton**
Changes:

1. I made `endpoint`, `updated_endpoint` and `graph` read-only attributes
2. `endpoint` is now a required positional argument
3. `graph` can now be left unspecified
4. I fixed the annotations that were wrong

Note about breaking changes. As long as I don't release the version 1.0.0, I allow myself breaking the API between each version (except patch version which is not used at this point most of the time).

When the first release will be done, compatibility will be maintain properly.

### CODE REVIEW

File: aiosparql/client.py [hunk 1]
```diff
@@ -80,19 +79,30 @@ def format_field(self, value, format_spec):
...
+        self._graph = graph
         self.session = aiohttp.ClientSession(**kwargs)
         self._generate_prefixes(prefixes)
 
+    @property
+    def endpoint(self):
+        return self._endpoint
+
+    @property
+    def update_endpoint(self):
```
Comment (asvetlov) [inline aiosparql/client.py]: Hmm. Sounds like a verb.
Verbs are used for methods usually, nouns for properties.
Maybe `endpoint_for_update` is better?

Comment (cecton) [inline aiosparql/client.py]: hmm... I never thought about that actually. I checked online to see how people call that and apparently the "SPARQL update endpoint" is valid.

https://etl.linkedpipes.com/components/l-sparqlendpoint.html

Comment (cecton) [inline aiosparql/client.py]: On the specs they just say:

> A service (often referred to by the informal term SPARQL endpoint) that accepts and processes update requests is referred to as an update service.

https://www.w3.org/TR/sparql11-update/#graphStoreQueryServices

Comment (asvetlov) [inline aiosparql/client.py]: Ok, let's follow status quo

**Pull request merged**